signac/__main__.py

```diff
@@ -1236,14 +1236,18 @@ def main():
         type=str,
```

Comment:
Can you add an example to the `parser_view` instance directly? Similar to line 1203.